app/src/main/java/com/zeapo/pwdstore/PasswordStore.kt

```diff
@@ -81,7 +80,7 @@ import org.eclipse.jgit.api.Git
 import org.eclipse.jgit.api.errors.GitAPIException
 import org.eclipse.jgit.revwalk.RevCommit
 
-class PasswordStore : AppCompatActivity(R.layout.activity_pwdstore) {
```

Comment:
I just noticed that this commit broke the "Back" button folder navigation and I think it's because of `BaseGitActivity`'s `onOptionsItemSelected` override. Maybe it's time to get rid of it?

Reply:
Yeah. I'll put up a PR.

Reply:
Pushed #1075